src/pages/selectSchedule/components/selectPriority/PriorityColumn.tsx

```diff
@@ -53,7 +66,7 @@ function PriorityColumn({ date, timeSlots }: ColumnStructure) {
             slotStyle={getPriorityColumntyle(selectedEntryId, priority)}
```

Comment:
위에서 언급한 변경사항에 맞게 여기도 이렇게 고쳐주시면 되겠습니다!!

```suggestion
            slotStyle={getPriorityColumnStyle(priority, selectedEntryId)}
```